tests/typecheck/contrib/sitemaps/test_generic_sitemap.yml

```diff
@@ -0,0 +1,49 @@
+-   case: test_items_custom_model
+    main: |
+        from django.contrib.sitemaps import Sitemap
+        from django.db.models import QuerySet
+        from django.urls import reverse
+        from myapp.models import Offer
+
+        class OfferSitemap(Sitemap):
```

Comment:
Do you mean 
```suggestion
        class OfferSitemap(Sitemap[Offer]):
```
?

Reply:
Yeah you're right. I just updated this and the other test.